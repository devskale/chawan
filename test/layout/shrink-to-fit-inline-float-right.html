<!DOCTYPE html>
<div style="display: inline-block">
<span>
test
<div style="float: right">right</div>
<div>flush</div>
test test test
<div style="float: right">right 2</div>
</div>
