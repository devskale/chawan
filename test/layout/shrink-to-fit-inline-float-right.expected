test             right
flush
test test test right 2
